manifests/init.pp

```diff
@@ -71,7 +71,7 @@
     ensure     => running,
     enable     => true,
     hasrestart => true,
-    restart    => '/usr/bin/systemctl reload nftables || /usr/bin/echo "#CONFIG BROKEN" >> /etc/nftables/puppet.nft && /bin/false',
+    restart    => '/usr/bin/systemctl reload nftables || ( /usr/bin/echo "#CONFIG BROKEN" >> /etc/nftables/puppet.nft && /bin/false)',
```

Comment:
maybe you need to wrapt it with `bash -c` since puppet might not run this as one to one in a shell

Reply:
This is correct now, it was very wrong logic before but now good.